vhdbuilder/packer/install-dependencies.sh

```diff
@@ -472,22 +472,27 @@ PATCHED_HYPERKUBE_IMAGES="
 1.15.11-hotfix.20200529.1
 1.15.11-hotfix.20200714.1
 1.15.11-hotfix.20200817.1
+1.15.11-hotfix.20200824.1
 1.15.12-hotfix.20200623.1
 1.15.12-hotfix.20200714.1
 1.15.12-hotfix.20200817.1
+1.15.12-hotfix.20200824.1
 1.16.9-hotfix.20200529.1
 1.16.10-hotfix.20200623.1
 1.16.10-hotfix.20200714.1
 1.16.10-hotfix.20200817.1
+1.16.10-hotfix.20200824.1
 1.16.13-hotfix.20200714.1
 1.16.13-hotfix.20200817.1
+1.16.13-hotfix.20200824.1
 1.17.3-hotfix.20200601.1
 1.17.7-hotfix.20200624
 1.17.7-hotfix.20200714.1
 1.17.7-hotfix.20200714.2
 1.17.7-hotfix.20200817.1
```

Comment:
we should not use `hyperkube:v1.17.7-hotfix.20200817.1` as this one has iptables issue

Reply:
If hyperkube v1.17.7 needs to be there, `v1.17.7-hotfix.20200714.1` does not have this issue

Reply:
deleted